### ISSUE TITLE
toml.parse_file does not panic when the file is missing

### ISSUE THREAD

**Issue by rodabt** [vlang/v#20911]
### Describe the bug

```v
import toml
toml.parse_file('file.toml') or { panic(err) }
```
if `file.toml` is not present, a panic should trigger, but it doesn't

### Reproduction Steps

In V repl:

```v
>>> import toml
>>> import rand
>>> filename := 'f_${rand.ulid()}.toml}' // random filename
>>> x :=  toml.parse_file(filename) or { panic(err) }
>>> println(x)
toml.Doc{
    ast: &ast.Root{
  input:  input.Config{
    text: ''
    file_path: ''
}
  table:  { }
}
}
```

### Expected Behavior

Should trigger a panic when file is missing

### Current Behavior

No errors. It should have fired a panic

### V version

Current V version: V 0.4.4 c6048d5, timestamp: 2024-02-26 21:05:38 +0200

### Environment details (OS name and version, etc.)

V full version: V 0.4.4 561fd94.c6048d5
OS: linux, Debian GNU/Linux 11 (bullseye) (WSL 2)
Processor: 8 cpus, 64bit, little endian, Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz

getwd: /tmp/test
vexe: /home/rabt/devel/v/v
vexe mtime: 2024-02-26 20:57:11

vroot: OK, value: /home/rabt/devel/v
VMODULES: OK, value: /home/rabt/.vmodules
VTMP: OK, value: /tmp/v_1000

Git version: git version 2.30.2
Git vroot status: weekly.2024.07-70-gc6048d50
.git/config present: true

CC version: cc (Debian 10.2.1-6) 10.2.1 20210110
thirdparty/tcc status: thirdparty-linux-amd64 12f392c3
> [!NOTE]
> You can use the 👍 reaction to increase the issue's priority for developers.
>
> Please note that only the 👍 reaction to the issue itself counts as a vote.
> Other reactions and those to comments will not be taken into account.

**Comment by billpcs** [comment]
Fixed at latest master with aa002a4019f3dc6d326f45c569cdc4d9b48ba934

**Comment by Delta456** [comment]
Closing
